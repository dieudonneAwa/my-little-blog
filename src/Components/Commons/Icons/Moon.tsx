import * as React from 'react';
import { Svg } from './ArrowBack';

function MoonIcon(props: React.SVGProps<SVGSVGElement>) {
  return (
    <Svg width={22} height={22} fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
      <path
        d="M21 11.9A10.022 10.022 0 1110.1 1 7.794 7.794 0 0021 11.9v0z"
        stroke="#000"
        strokeWidth={2}
        strokeLinecap="round"
        strokeLinejoin="round"
      />
    </Svg>
  );
}

export default MoonIcon;
