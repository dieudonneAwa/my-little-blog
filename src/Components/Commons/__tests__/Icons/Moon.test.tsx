/* eslint-disable import/no-extraneous-dependencies */
import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import Moon from '../../Icons/Moon';

describe('Moon', (): void => {
  let wrapper: ShallowWrapper<void>;
  beforeEach((): void => {
    wrapper = shallow(<Moon />);
  });

  it('should render Moon component', (): void => {
    expect(wrapper).toMatchSnapshot();
  });
});
