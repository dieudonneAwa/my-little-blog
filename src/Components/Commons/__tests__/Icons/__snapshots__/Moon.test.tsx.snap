// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Moon should render Moon component 1`] = `
<svg
  fill="none"
  height={22}
  width={22}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M21 11.9A10.022 10.022 0 1110.1 1 7.794 7.794 0 0021 11.9v0z"
    stroke="#000"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth={2}
  />
</svg>
`;
